crates/swc_ecma_minifier/tests/terser/compress/evaluate/pow_nan/input.js

```diff
@@ -0,0 +1 @@
+console.log(Math.pow(1, NaN));
```

Comment:
Please add another test case:

```
console.log(Math.pow(NaN, 0));
```

Reply:
sure

Reply:
I added a condition branch for number to the power of 0, which should equal to 1. Is that ok?